desc/compute/_curve.py

```diff
@@ -1061,9 +1061,7 @@ def _torsion(params, transforms, profiles, data, **kwargs):
     coordinates="",
     data=["ds", "x_s"],
     parameterization=[
-        "desc.geometry.curve.FourierRZCurve",
-        "desc.geometry.curve.FourierXYZCurve",
-        "desc.geometry.curve.FourierPlanarCurve",
+        "desc.geometry.core.Curve",
```

Comment:
I'm not going to make you change it, but the unnecessary comma really bothers me. Why use 3 line of code when it could fit on a single line?!